src/extendedstorage.cpp

```diff
@@ -479,7 +391,7 @@ bool ExtendedStorage::updateNotebook(const Notebook::Ptr &nb)
         return false;
     }
 
-    if (!nb->isRunTimeOnly() && !modifyNotebook(nb, DBUpdate)) {
+    if (!nb->isRunTimeOnly() && !modifyNotebook(nb)) {
```

Comment:
Maybe updateNotebook() as name could be slightly clearer since the operation still maps to DBUpdate on lower level.

Reply:
The problem is that `addNotebook(Notebook::Ptr&)`,  `updateNotebook(Notebook::Ptr&)` and  `deleteNotebook(Notebook::Ptr&)` are already part of the public API with another meaning...

Reply:
Ah, of course.